### PR TITLE
Add feature of unwanted contributions

### PR DESCRIPTION
**Pull request by kuanhanl**
The feature of unwanted contributions is based on Weifeng Chen, Pro. Biegler, and Dr. Garcia's paper published in 2019. It offers a good way to deal with the time-variant or time-invariant unwanted contributions in the measured spectra data. Two examples of each type of unwanted contributions are provided and one example of multiple experiments with different types of unwanted contributions is included as well! The tutorial of this new feature is updated in the documentation files. Also, tests in validate_tut_problems.py and validation_script.py all passed.

In addition, some bugs in MultipleExperimentsEstimator.py were found and fixed. I already checked the results of previous examples related to this package and they all worked fine.

I know Kevin almost finished his work and I hope the merge would be ok for two pull requests.

### CODE REVIEW

File: kipet/examples/Ex_15_estimation_mult_exp_unwanted_G.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+#
+#		\frac{dZ_a}{dt} = -k_1*Z_a	                Z_a(0) = 1
+#		\frac{dZ_b}{dt} = k_1*Z_a - k_2*Z_b		Z_b(0) = 0
+#               \frac{dZ_c}{dt} = k_2*Z_b	                Z_c(0) = 0
+#               C_k(t_i) = Z_k(t_i) + w(t_i)    for all t_i in measurement points
+#               D_{i,j} = \sum_{k=0}^{Nc}C_k(t_i)S(l_j) + \xi_{i,j} for all t_i, for all l_j 
+#       Initial concentration 
+
+from __future__ import print_function
+from kipet.library.TemplateBuilder import *
```
Comment (kwmcbride) [inline kipet/examples/Ex_15_estimation_mult_exp_unwanted_G.py]: I think we should move away from using global imports. It is generally frowned upon.

**Comment by mchlshort** [comment]
Well done, Kuan-Han! I really like what you have done here. Seems to work really well.

**Review by mchlshort — APPROVED** [review]
Great work, @kuanhanl! I can't see any issues and it seems to pass all tests this side.

**Comment by kuanhanl** [comment]
Thank you, @mchlshort. I am happy you like it!

**Pull request merged**